### PR TITLE
Fixed readme typo

### PR DESCRIPTION
**Pull request by mainrs**

### CODE REVIEW

**Comment by Forecaster** [comment]
You are required to sign the Contributor License Agreement before this pull request can be accepted.

**Review by Forecaster — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The mod was created and is still currently maintained by the user going by the n
...
+> As a new modder, I originally feared losing control of my code, my brainchild. However, since that time, I have had the privelege of being Project Lead on two other major Minecraft Mod projects that provided access to the source: [Buildcraft](https://github.com/BuildCraft/BuildCraft) and [Forestry](https://github.com/ForestryMC/ForestryMC). I've generally found this to be a positive experience resulting in many bug fixes and increased intermod compatibility.  While I still have some concerns, I have come to feel that the benefits of providing others access to my source code outweighs the negatives and unknowns. To that end, despite my missgivings, I made Source Access a [Patreon Milestone Goal](http://www.patreon.com/CovertJaguar). I had no idea whether I'd ever meet that Goal, but I decided to let the community decide, and decide they did, overwhelminging so! My Patrons are awesome. So, as promised, I am posting the Source Code.
```
Comment (liach) [inline README.md]: `missgivings` should be `misgivings`

**Review by liach — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mainrs) [reply README.md]: Will fix that too then.

**Review by liach — APPROVED** [review]

**Comment by liach** [comment]
@Forecaster Next time please squash merge such changes (too little changes)

**Pull request merged**